### PR TITLE
Show stargazers you know

### PR DESCRIPTION
**Pull request by Hermanya**
image
This is what it normally looks like.

---

image
This is what it looks like on hover.

---

When there is no people to show, nothing is shown.

### CODE REVIEW

**Comment by Hermanya** [comment]
Not quite sure how I broke the build.
```
> xo && stylelint source/*.css
/home/travis/build/sindresorhus/refined-github/node_modules/nanomatch/lib/compilers.js:213
      var before = prev.prev || {};
                        ^
TypeError: Cannot read property 'prev' of undefined

```

**Comment by sindresorhus** [comment]
I don't think this is important enough to be in the UI by default. It will make the repo view very noisy. I think it would be better to instead show it when you hover the Star button.

File: source/features/show-stargazers-you-know.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import {h} from 'dom-chef';
+import select from 'select-dom';
+import domify from '../libs/domify';
+import {getRepoURL} from '../libs/page-detect';
+
+const extractUserData = element => {
+	const image = element.querySelector('img');
+	const link = element.querySelector('a').href;
+	return {
+		avatar: image.src.replace(/\?.+/, ''),
```
Comment (sindresorhus) [inline source/features/show-stargazers-you-know.js]: Better to use `new URL(image.src)` and properly pull out or remove what you want.

**Comment by bfred-it** [comment]
> show it when you hover the Star button

Even then I don't know where it would make sense to show it.

**Comment by Hermanya** [comment]
@sindresorhus perhaps I did not properly state my case. This is what it would look like; imho does not seem to make repo that much noisy.
image

The reason why I think it's important is because social proof is a useful tool in evaluating a project. If I stumble upon a project does not have many stars, but one of my friends starred it, it gives the project some credibility. I could ask that friend why they starred the project; what made it interesting for them; did they have a chance to use the project.

I agree that showing all avatars by default would make the page look noisy. Which is why it only shows 2 avatars at most, and a stack graphic to indicate that there is more.

The reason, why I think it should not be hidden behind the star button, is because nobody would ever look there. It's the same reason why many people don't know about /stargazers/you_know page.

Eventually, it would be cool, if when you visit a repo, it would show you all activity from people you know. Such as "Alex starred this repo", "Jane submitted a pull request for _", "Mary has pushed daily for the past _ days", "George used this package on _ project". 

Let's make GitHub more social 😄

**Comment by sindresorhus** [comment]
I agree with your reasoning for why it's useful, I just don't think it's useful *enough* to be placed there.

**Comment by Hermanya** [comment]
I've been using this feature for the past couple of weeks, and I find it pretty useful.
For one thing, when I find an interesting project, I can immideately see if I'm the first one, amongst my friends, to find it. I know it sounds silly, but I enjoy the feeling.
And another use case I found, is when I go through my news feed, I open interesting projects in new tabs. Then when I eventually open that tab, I lose the context, of why this repo is open. If I see one avatar, it's a good indicator that that person starred the project, and it appeared in my news feed. By news feed I refer to the root page.
I usually can associate avatar to a person. And if I cannot, then I click on the avatar to see their profile. And if I don't know why I'm following that person, I unfollow them.

**Comment by sindresorhus** [comment]
I'm going to close this as the PR in its current form is too intrusive per my previous comments, and it doesn't look like the author is interested in changing it. I'm open to explore adding other social features to Refined GitHub.

**Pull request closed without merging**